Nestle1904/lowfat/treedown.css

```diff
@@ -217,3 +217,15 @@ wg[rule='sub-CL']:not(> *[role])::before {
       and the role of the sub-CL wg. E.g. διότι in Luke 2.7 */
   position: absolute;
 }
+
+/* Ryder: This rule ensures nominalized clauses are properly indented below their nominalizing NP */
+wg > wg[clausetype='nominalized-clause'] {
+  margin-left: 1rem;
+}
+
+/* Ryder: This rule rule adds additional margin when a nominalized clause is part of apposition, because the nominalizing article has a 'display: block' property, which means it will not display the subordination properly. Like the other CSS rules that rely on Node rules (i.e., 'Np-Appos'), this will have to be cleaned up when we add an apposition attribute to the lowfat. */
+wg[rule='Np-Appos']
```

Comment:
I can update this line actually now that [this pr](https://github.com/Clear-Bible/macula-greek/pull/60) is closed